lib/widget/lottie/lottie.dart

```diff
@@ -28,7 +30,66 @@ class EnsembleLottie extends StatefulWidget
 
   @override
   Map<String, Function> methods() {
-    return {};
+    if (kIsWeb) {
+      // A little hacky way to check if html renderer is used as only html render would have the lottieController null.
+      final bool isHtml = _controller.lottieController != null;
```

Comment:
yes this is a really hacky way. Also I see that you use _controller.lottieController! everywhere.

Reply:
@hemish11 this is just super hacky, please make it cleaner so anyone working on the code later can make sense of it

Reply:
The simpler way that I first tried to do was to check by this condition: `js.context['flutterCanvasKit'] != null;` but for this, import `'dart:js'` as js; needs to be imported in the main lottie.dart file which is not possible as the app starts to crash when importing the web-only library when it runs on mobile. It has been imported in web/lottiestate.dart but as it only runs when it is in web there is no issue with that import. Also currently I'm adding more comments in the code so that its easy to understand.